//
// protoc --go_out=. EncodingConfig.proto
//

syntax = "proto3";

package sotcore;

option csharp_namespace = "CloudSync.Core";

message EncodingConfig {
	int32 compression_level = 1;
    repeated string ciphers = 2;
	bytes key = 3; // All keys are passed as a single array

	string src = 4;
	string dest = 5;
}